aws/resource_aws_cloudsearch_service.go

```diff
@@ -0,0 +1,471 @@
+package aws
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudsearch"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsCloudSearchDomain() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsCloudSearchDomainCreate,
+		Read:   resourceAwsCloudSearchDomainRead,
+		Update: resourceAwsCloudSearchDomainUpdate,
+		Delete: resourceAwsCloudSearchDomainDelete,
+
+		Schema: map[string]*schema.Schema{
+			"domain_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"instance_type": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
Could you point me to the documentation you used for this parameters?
Only found http://docs.aws.amazon.com/cloudsearch/latest/developerguide/API_ScalingParameters.html, but nothing being required :/